packaging/version.sh

```diff
@@ -16,7 +16,8 @@ build=""
 
 if [ ! "$mode" = "production" ] ; then
   current_date="$(date -u +%Y%m%d%H%M%S)" # UTC
-  commit_short="$(git -C "$here" log -1 --pretty=format:%h)"
+  commit_short="$(git -C "$here" log -1 --pretty=format:%h || \
+    echo -n ${SOURCE_COMMIT:0:10})"
```

Comment:
?

Reply:
dockerignore skips `.git` so all the git commands won't work, SOURCE_COMMIT is pushed into the container using docker build args

ignore is done to reduce the build times ~3x